### ISSUE TITLE
Failed to serialize on MonaUdsClient.export when MonaSingleMessage is used in multiclass inheritance

### ISSUE THREAD

**Issue by RafaAguilar** [monalabs/mona-uds-client#12]
# Symptom

`MonaUdsClient.export` fail to serialize a timestamp field which is not part of the `MonaSingleMessage` but part of mixin:

```
Mona failed when trying to serialize data: Cannot serialize datetime.datetime(2021, 5, 3, 22, 32, 37, 78834, tzinfo=datetime.timezone.utc)
```

**Note**: this would happen for any non-serializable attribute.

# How to Reproduce
Minimum sample:
```python
#!/usr/bin/env python3
from abc import ABC
from datetime import datetime, timezone
from dataclasses import dataclass

from mona_uds_client.mona_uds_client import MonaSingleMessage, MonaUdsClient

@dataclass
class BaseMessage(ABC):
   timestamp: datetime
   epoch = datetime(1970, 1, 1, tzinfo=timezone.utc)


class MonaMessage(BaseMessage, MonaSingleMessage):
    def __init__(self,
        timestamp: datetime = datetime.now(timezone.utc),
    ):

        BaseMessage.__init__(
            self,
            timestamp=timestamp
        )

        MonaSingleMessage.__init__(
            self,
            contextId="1231231",
            message={"field1": "blah"},
            arcClass="test_class",
            exportTimestamp=int((self.timestamp - self.epoch).total_seconds()),
        )


if __name__ == "__main__":
    message = MonaMessage()
    client = MonaUdsClient()
    
    client.export([message])
```
It should raise an exception with a similar text like below:
```bash
Cannot serialize datetime.datetime(2021, 5, 3, 23, 15, 19, 262444, tzinfo=datetime.timezone.utc)
```

Note: `SHOULD_RAISE_EXCEPTIONS` should bet set to True, otherwise jsut retuns `False`.

# Root cause

The datetime field from the `BaseClass` named `timestamp` is getting serialized but it does not belong to the `MonaSingleMessage`. Check [Line 195](https://github.com/monalabs/mona-uds-client/blob/a7c42ad315a69519098857d6248008a3224ae75a/mona_uds_client/mona_uds_client.py#L195).

# Expected Behavior

Serialize and export the message successfully including only the fields declared in `MonaSingleMessage`.

**Comment by RafaAguilar** [comment]
This [PR](https://github.com/monalabs/mona-uds-client/pull/13) fixes it, however it might need to be adapted (e.g. formatting, testing...)

**Comment by RafaAguilar** [comment]
Fixed through #14 for better encapsulation.
